### PR TITLE
Facia CSS and icon refactor

### PR DESCRIPTION
**Pull request by phamann**
This is the last major chunk of performance work for Facia. It's quite large but unfortunately has to go in one piece. Sorry :crying_cat_face: 
### Changes:
- Do not load global.css on facia pages
- Create project specific CSS file `facia.css`
- Inline all navigation icons to SVG
- Inline all facia container icons (such as show more)
- Refactor comment-count and show-more JS with new inline icons
- Fix layout of slim nav with new inline icons
### TODO:
- Remove legacy icons from the global icon folder to prevent being compiled into global.css
- Rinse and repeat for content pages
### Tests

http://www.browserstack.com/screenshots/708e9ad04d39f55c8cc7837a5769871c4eeeee32

/cc @sndrs @robertberry @michaelwmcnamara

### CODE REVIEW

File: common/app/views/fragments/header.scala.html [hunk 1]
```diff
@@ -67,21 +67,21 @@
                     @if(metaData.section == "observer" && metaData.isFront) {
                         <a href="@LinkTo{@metaData.url}" data-link-name="site logo" id="logo" class="logo-wrapper logo-wrapper--observer" itemprop="url" data-component="logo">
                             @if(metaData.hasSlimHeader) {
-                                <i class="i i-observer-logo-160"></i>
+                                @fragments.inlineSvg("observer-logo-160", "logo")
                             } else {
                                 <span class="u-h" itemprop="name">The Observer</span>
-                                <i class="i i-observer-logo-160 mobile-only"></i>
-                                <i class="i i-observer-logo-320 hide-on-mobile"></i>
+                                @fragments.inlineSvg("observer-logo-160", "logo", List("mobile-only"))
```
Comment (sndrs) [inline common/app/views/fragments/header.scala.html]: do we need 2 svgs here?

File: common/app/views/fragments/header.scala.html [hunk 2]
```diff
@@ -67,21 +67,21 @@
...
                     } else {
                         <a href="@LinkTo{/}" data-link-name="site logo" id="logo" class="logo-wrapper" itemprop="url" data-component="logo">
                             <span class="u-h" itemprop="name">The Guardian</span>
                             @if(metaData.hasSlimHeader) {
-                                <i class="i i-guardian-logo-160"></i>
+                                @fragments.inlineSvg("guardian-logo-160", "logo")
                             } else {
-                                <i class="i i-guardian-logo-160 mobile-only"></i>
-                                <i class="i i-guardian-logo-320 hide-on-mobile"></i>
+                                @fragments.inlineSvg("guardian-logo-160", "logo", List("mobile-only"))
```
Comment (sndrs) [inline common/app/views/fragments/header.scala.html]: again, only 1 needed?

File: static/src/inline-svgs/icon/arrow-left.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="6" height="12" viewBox="0 0 6 12"><path d="M6,11.5 L1.5,6 L6,0.5 L5.5,0 L0,5.75 L0,6.25 L5.5,12 L6,11.5 L6,11.5 Z"/></svg>
```
Comment (sndrs) [inline static/src/inline-svgs/icon/arrow-left.svg]: `xmlns` will knacker IE8  - http://stuntbox.com/blog/2013/06/bulletproof-inline-svg

File: static/src/inline-svgs/icon/external-link.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="10" viewBox="0 0 14 10"><g><path d="M12 1.1L8.27 4.83l.902.9L13 2.07 13.2 5h.8V.4l-.4-.4H9v.8zM11.7 6.625V8.7H1.2V2.2h5l.8-.6V1H.6l-.6.6v7.8l.6.6h11.8l.6-.6V6h-.7z"/></g></svg>
```
Comment (sndrs) [inline static/src/inline-svgs/icon/external-link.svg]: don't need the `g`?

File: static/src/inline-svgs/icon/minus.svg [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
+    <g stroke="none" stroke-width="1">
```
Comment (sndrs) [inline static/src/inline-svgs/icon/minus.svg]: could just be a `line` coudln't it?

File: common/app/views/fragments/header.scala.html [hunk 6]
```diff
@@ -67,21 +67,21 @@
...
+                                @fragments.inlineSvg("observer-logo-160", "logo", List("mobile-only"))
+                                @fragments.inlineSvg("observer-logo-320", "logo", List("hide-on-mobile"))
                             }
                         </a>
                     } else {
                         <a href="@LinkTo{/}" data-link-name="site logo" id="logo" class="logo-wrapper" itemprop="url" data-component="logo">
                             <span class="u-h" itemprop="name">The Guardian</span>
                             @if(metaData.hasSlimHeader) {
-                                <i class="i i-guardian-logo-160"></i>
+                                @fragments.inlineSvg("guardian-logo-160", "logo")
```
Comment (marchibbins) [inline common/app/views/fragments/header.scala.html]: Same old story, `identityHeader` too please!

File: static/src/stylesheets/module/facia/_icons.scss [hunk 7]
```diff
@@ -2,16 +2,15 @@
...
+    .inline-icon {
+        fill: colour(news);
         position: relative;
         height: .7em;
         width: 1em;
     }
 
-    &:hover:after {
-        @include icon(arrow-light-blue-right, false);
+    &:hover .inline-icon {
```
Comment (sndrs) [inline static/src/stylesheets/module/facia/_icons.scss]: pretty sure you can leave out the `.inline-icon` specification, it should just inherit?

File: common/app/views/fragments/header.scala.html (hunk 6 above)
Comment (phamann) [inline common/app/views/fragments/header.scala.html]: @marchibbins This does affect the identity header as it has its own template and CSS file. I'd prefer to leave and then do the work to replace the identity header with the new slim nav.

Comment (marchibbins) [inline common/app/views/fragments/header.scala.html]: No worries, so long as the logo doesn't disappear from the sprite before then.

File: static/src/stylesheets/module/facia/_icons.scss [hunk 8]
```diff
@@ -138,66 +118,59 @@
...
+    .fc-sublink--gallery .fc-sublink__link .inline-icon {
+        fill: #000000;
     }
 
-    .fc-sublink--audio .fc-sublink__link:before,
-    .fc-sublink--audio.tone-podcast--sublink .fc-sublink__link:before {
-        @include icon(volume-high-black, false);
+    .fc-sublink--audio .fc-sublink__link path,
+    .fc-sublink--audio.tone-podcast--sublink .fc-sublink__link .inline-icon {
+        fill: #000000;
```
Comment (sndrs) [inline static/src/stylesheets/module/facia/_icons.scss]: can the `fill` inherit from `.fc-sublink--audio.tone-podcast--sublink`, save a few selectors?

**Comment by sndrs** [comment]
:+1: apart from the nitpicking. nice one

File: static/src/stylesheets/module/facia/_icons.scss (hunk 7 above)
Comment (phamann) [inline static/src/stylesheets/module/facia/_icons.scss]: As discussed this is needed due to the parent selector being used.

File: static/src/inline-svgs/icon/arrow-up.svg [hunk 9]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 18"><path d="M.4 15.3l10.5-8.4 1.1-.9 1.1.9 10.5 8.4-.5.7-11.1-6.3-11.1 6.3-.5-.7z"/></svg>
```
Comment (austinh7) [inline static/src/inline-svgs/icon/arrow-up.svg]: Do you not need to remove xmlns from these for IE8?

See @sndrs doc here, https://github.com/guardian/frontend/blob/master/docs/inline-svgs.md

**Pull request merged**